### PR TITLE
Added step to open custom options page

### PR DESCRIPTION
**Pull request by SIGEER2222**
I found this step missing in newer versions of MSDN searches, so it's bothering me a bit, I'm hoping to add it here or somewhere else

### CODE REVIEW

**Comment by Jak-MS** [comment]
@SIGEER2222 - review of your changes is pending until you sign the CLA. Thanks!


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/extensibility/creating-an-options-page.md [hunk 1]
```diff
@@ -265,6 +265,29 @@ This walkthrough creates a simple Tools/Options page that uses a property grid t
 
      A message box displays the current value of `OptionInteger`.
 
+## Open options page
+
+In this section, you will add you will add a Command and an event for the button to open the options page
```
Comment (ghogen) [inline docs/extensibility/creating-an-options-page.md:270]:
```suggestion
In this section, you'll add a Command and an event for the button to open the options page
```

File: docs/extensibility/creating-an-options-page.md [hunk 2]
```diff
@@ -265,6 +265,29 @@ This walkthrough creates a simple Tools/Options page that uses a property grid t
...
+
+1. First add a Comman called OpenpageCommand.cs.
```
Comment (ghogen) [inline docs/extensibility/creating-an-options-page.md:272]:
```suggestion
1. First add a file called OpenPageCommand.cs. 
```

File: docs/extensibility/creating-an-options-page.md [hunk 3]
```diff
@@ -265,6 +265,29 @@ This walkthrough creates a simple Tools/Options page that uses a property grid t
 
      A message box displays the current value of `OptionInteger`.
 
+## Open options page
+
+In this section, you will add you will add a Command and an event for the button to open the options page
+
+1. First add a Comman called OpenpageCommand.cs. 
+
+3. Then open OpenpageCommand.cs and change your Execute method.
```
Comment (ghogen) [inline docs/extensibility/creating-an-options-page.md:274]:
```suggestion
3. Then, open *OpenPageCommand.cs* and change your Execute method.
```

File: docs/extensibility/creating-an-options-page.md [hunk 4]
```diff
@@ -265,6 +265,29 @@ This walkthrough creates a simple Tools/Options page that uses a property grid t
...
+         Type optionsPageType = typeof(OptionPageCustom);
+         Instance.package.ShowOptionPage(optionsPageType);
+     }
+    ```
+
+3. Run the project, then click the Invoke XXX button (it is under the Tool option by default), then you can see your options page is opened
+
+5. More details about opening options page can refer to the following documents
+
+- [Opening an Options Page](https://docs.microsoft.com/en-us/previous-versions/cc826083(v=vs.140))
```
Comment (ghogen) [inline docs/extensibility/creating-an-options-page.md:289]:
```suggestion
- [Opening an Options Page](/previous-versions/cc826083(v=vs.140))
```

File: docs/extensibility/creating-an-options-page.md [hunk 5]
```diff
@@ -265,6 +265,29 @@ This walkthrough creates a simple Tools/Options page that uses a property grid t
...
+     ```csharp
+     private void Execute(object sender, EventArgs e)
+     {
+         ThreadHelper.ThrowIfNotOnUIThread();
+         Type optionsPageType = typeof(OptionPageCustom);
+         Instance.package.ShowOptionPage(optionsPageType);
+     }
+    ```
+
+3. Run the project, then click the Invoke XXX button (it is under the Tool option by default), then you can see your options page is opened
```
Comment (ghogen) [inline docs/extensibility/creating-an-options-page.md:285]:
```suggestion
3. Run the project, then click the Invoke button (it is under the Tool option by default), then you can see your options page is opened.
```

**Comment by ghogen** [comment]
@leslierichardson95 @SIGEER2222 
Thanks for the submission. I reviewed and made changes for correctness, validation issues, and typos, etc., but @lelierichardson would need to review to confirm that this is a good idea. I don't know why this section was previously removed. If there was a good reason, there might be a problem with adding it back.

**Comment by SIGEER2222** [comment]
It's always right to be careful
and&nbsp;thanks for the correction, I'm too careless hhh




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "MicrosoftDocs/visualstudio-docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 凌晨0:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [MicrosoftDocs/visualstudio-docs] Added step to open custom options page (PR #8362)





 
@leslierichardson95 @SIGEER2222
 Thanks for the submission. I reviewed and made changes for correctness, validation issues, and typos, etc., but @lelierichardson would need to review to confirm that this is a good idea. I don't know why this section was previously removed. If there was a good reason, there might be a problem with adding it back.

**Review by leslierichardson95 — APPROVED** [review]

**Comment by j-martens** [comment]
#sign-off

**Comment by j-martens** [comment]
@leslierichardson95 can you sign off please? thanks.

**Comment by leslierichardson95** [comment]
#sign-off

**Comment by jborsecnik** [comment]
@SIGEER2222 - review of your changes is pending until you sign the CLA. It's all electronic and will take just a few minutes: https://cla.microsoft.com. 

 Thanks!

**Comment by SIGEER2222** [comment]
@jborsecnik Sorry to reply so late, I have already submitted a signature request

**Review by SIGEER2222 — COMMENTED** [review]
Better version

**Comment by j-martens** [comment]
#sign-off

**Pull request merged**